runtime/legion/legion_tasks.cc

```diff
@@ -5760,6 +5912,13 @@ namespace Legion {
     void PointTask::perform_versioning_analysis(std::set<RtEvent> &ready_events)
     //--------------------------------------------------------------------------
     {
+      if (memoizing)
```

Comment:
I feel like all these places where you have 'if (memoizing)' need to be captured by the Memoizable interface. We should talk about how to do this.